### PR TITLE
[MOSTLY MODULAR] Central Command Sabre (Also for NTC)

### PR DESCRIPTION
**Pull request by ChromeFoxxity**
## About The Pull Request

Makes some additions within a few NT Rep Offices, as well as adds the CentCom sabre to said NT Rep's locker. Does less force than the average captain's sabre. As well as an addition of a few cosmetic pieces into their garment bag.

## Why It's Good For The Game

It's purely cosmetic, and maybe a bit of a balance change with the new sword. But I'm sure it'll be fine considering you can already buy one from Cargo.

## Proof Of Testing

This is untested due to Visual Studio Code breaking every time I try to run it. I apologize, but I'm sure it is working perfectly.
image

## Changelog

:cl:
add: Official's suit to the Representative's garment bag.
add: CentCom sword sabre to the Representative's locker.
add: A few more additions to some of the Representative's offices.
balance: Added a shift-start sword for the Representative.
image: Added modular icons for sword and sheath.
code: Added modular code for sword and sheath.
/:cl:

image

### CODE REVIEW

**Comment by ReturnToZender** [comment]
I think your branch is super out of date with Master, you have a bunch of map changes that don't really have anything to do with your PR

**Comment by ChromeFoxxity** [comment]
> I think your branch is super out of date with Master, you have a bunch of map changes that don't really have anything to do with your PR
> 
> Also the block chance and armor penetration need to be reduced, the cargo sabre has 20 and 25, the captain's saber has 50 and 75, to give you a reference of where the values need to be at most

I'll be honest here, I also think it may be outdated. My branches got a little screwed up. Let me try fixing it.

**Comment by ChromeFoxxity** [comment]
> I think your branch is super out of date with Master, you have a bunch of map changes that don't really have anything to do with your PR
> 
> Also the block chance and armor penetration need to be reduced, the cargo sabre has 20 and 25, the captain's saber has 50 and 75, to give you a reference of where the values need to be at most

Also, what do you mean several map changes? I've only changed 15 files, from what the github says.

**Comment by ChromeFoxxity** [comment]
> > > I think your branch is super out of date with Master, you have a bunch of map changes that don't really have anything to do with your PR
> > > Also the block chance and armor penetration need to be reduced, the cargo sabre has 20 and 25, the captain's saber has 50 and 75, to give you a reference of where the values need to be at most
> > 
> > 
> > Also, what do you mean several map changes? I've only changed 15 files, from what the github says. And, those map changes are additions to the offices as I've said.
> 
> You misunderstood, they meant that the changes in the automapper, some of them do not line up and appear to be out of date

Ah, I see.. I think it is unfortunately broken, yes.

**Comment by ChromeFoxxity** [comment]
> > > I think your branch is super out of date with Master, you have a bunch of map changes that don't really have anything to do with your PR
> > > Also the block chance and armor penetration need to be reduced, the cargo sabre has 20 and 25, the captain's saber has 50 and 75, to give you a reference of where the values need to be at most
> > 
> > 
> > Also, what do you mean several map changes? I've only changed 15 files, from what the github says. And, those map changes are additions to the offices as I've said.
> 
> You misunderstood, they meant that the changes in the automapper, some of them do not line up and appear to be out of date

Is there a way to fix this swiftly? I don't think I can do it in this same PR.

**Comment by ChromeFoxxity** [comment]
> > > > I think your branch is super out of date with Master, you have a bunch of map changes that don't really have anything to do with your PR
> > > > Also the block chance and armor penetration need to be reduced, the cargo sabre has 20 and 25, the captain's saber has 50 and 75, to give you a reference of where the values need to be at most
> > > 
> > > 
> > > Also, what do you mean several map changes? I've only changed 15 files, from what the github says. And, those map changes are additions to the offices as I've said.
> > 
> > 
> > You misunderstood, they meant that the changes in the automapper, some of them do not line up and appear to be out of date
> 
> Is there a way to fix this swiftly? I don't think I can do it in this same PR.

Actually, wait. Give me a moment, I'm gonna try to rebase it.

**Comment by YakumoChen** [comment]
I'm pretty skeptical why they need a sword in the first place

Sure they can buy one from cargo but we've already made a point of disarming their roundstart weaponry before.

**Comment by ChromeFoxxity** [comment]
> I'm pretty skeptical why they need a sword in the first place
> 
> Sure they can buy one from cargo but we've already made a point of disarming their roundstart weaponry before.

It's for formality, it isn't THAT bad, it might be even worse than the one from cargo if it needs be.

**Comment by YakumoChen** [comment]
Sabres as decoration is something usually given to military officers or similarly high authority, of which the Consultant (not rep, the NTC is not a rep) is neither

**Comment by ChromeFoxxity** [comment]
> Sabres as decoration is something usually given to military officers or similarly high authority, of which the Consultant (not rep, the NTC is not a rep) is neither

They have alternate titles of "Representative", their office is also named "ntrep's office", in code. They are a representative, for all I could care.

**Comment by IgiariValkyr** [comment]
Add it anyways for the admins and their admiral characters. This is a threat. Here's a piece of paper with a gun drawn on it to threaten you to adding it anyways.
maxresdefault

**Comment by YakumoChen** [comment]
> > Sabres as decoration is something usually given to military officers or similarly high authority, of which the Consultant (not rep, the NTC is not a rep) is neither
> 
> They HAD alternate titles of "Representative", their office is also named "ntrep's office", in code. They are also just as luxurious as the Captain when it comes to their clothes, they might be intern-level, but this is pretty much just for luxury.

HAD is a key word. They aren't reps, they have no authority, they're corporate lawyers with bridge access.

**Comment by ChromeFoxxity** [comment]
> > > Sabres as decoration is something usually given to military officers or similarly high authority, of which the Consultant (not rep, the NTC is not a rep) is neither
> > 
> > 
> > They HAD alternate titles of "Representative", their office is also named "ntrep's office", in code. They are also just as luxurious as the Captain when it comes to their clothes, they might be intern-level, but this is pretty much just for luxury.
> 
> HAD is a key word. They aren't reps, they have no authority, they're corporate lawyers with bridge access, and they don't need a sword. They shouldn't be treated like decorated officers.

They quite literally have a heavy rifle, like the Captain. Besides, they're basically treated like representatives. I don't know what your issue is with this. It's barely gonna change anything due to the fact every NTC buys a sword.

**Comment by projectkepler-ru** [comment]
> > Sabres as decoration is something usually given to military officers or similarly high authority, of which the Consultant (not rep, the NTC is not a rep) is neither
> 
> They HAD alternate titles of "Representative", their office is also named "ntrep's office", in code. They are also just as luxurious as the Captain when it comes to their clothes, they might be intern-level, but this is pretty much just for luxury.

this point is moot, file name for things are usually there for convenient sake rather than to be taken literally

File: modular_zubbers/code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/obj/item/storage/belt/sabre/centcom
+	name = "commander's sabre sheath"
+	desc = "An incredibly ornate sabre sheath, meant to hold a commander's sabre. The cloth it's made of is soft to the touch, whilst the gold is also of incredibly purity while also alloyed with plasteel."
+	icon = 'modular_zubbers/icons/obj/clothing/belts.dmi'
+	icon_state = "cc_sheath"
+	inhand_icon_state = "cc_sheath"
+	worn_icon_state = "cc_sheath"
```
Comment (projectkepler-ru) [inline modular_zubbers/code/game/objects/items/storage/belt.dm:7]: This is failing the worn icon test please re-check that your file are working correctly

**Comment by projectkepler-ru** [comment]
> > > Sabres as decoration is something usually given to military officers or similarly high authority, of which the Consultant (not rep, the NTC is not a rep) is neither
> > 
> > 
> > They HAD alternate titles of "Representative", their office is also named "ntrep's office", in code. They are also just as luxurious as the Captain when it comes to their clothes, they might be intern-level, but this is pretty much just for luxury.
> 
> HAD is a key word. They aren't reps, they have no authority, they're corporate lawyers with bridge access, and they don't need a sword. They shouldn't be treated like decorated officers.



they can have it, not a huge loss, the sword stats quite weak anyway

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]
If the failing unit test can be resolved It'll get merged

**Pull request closed without merging**